pkgs/applications/blockchains/snarkvm/default.nix

```
@@ -0,0 +1,48 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "snarkVM";
```

`pname` should be all lowercase
```suggestion
  pname = "snarkvm";
```